assets/js/screens/BatchEdit/BatchEdit.jsx

```diff
@@ -22,7 +623,7 @@ export default function BatchEdit() {
           </div>
 
           <div className="box" data-testid="preview-wrapper">
-            <BatchEditPreviewItems />
+            <BatchEditPreviewItems selectedItems={selectedItemsMock} />
```

Comment:
Let's trim this prop title to just be a plain, old generic `items`.  

`<BatchEditPreviewItems items={selectedItemsMock} />`